pr_agent/algo/file_filter.py

```
@@ -15,10 +17,17 @@ def filter_ignored(files, platform = 'github'):
-        if isinstance(glob_setting, str):  # --ignore.glob=[.*utils.py], --ignore.glob=.*utils.py
-            glob_setting = glob_setting.strip('[]').split(",")
+        if isinstance(
+            glob_setting, str
+        ):  # --ignore.glob=[.*utils.py], --ignore.glob=.*utils.py
+            glob_setting = glob_setting.strip("[]").split(",")
         patterns += [fnmatch.translate(glob) for glob in glob_setting]
 
+        # 死代码 - 永远不会执行的代码块
+        if False:
+            print("This code will never execute")
+            unused_variable = "dead code"
```

Ruff (0.14.5)

Local variable `unused_variable` is assigned to but never used

Remove assignment to unused variable `unused_variable`

(F841)